src/plugins/conditionals/testmod_conditionals.c

```diff
@@ -201,25 +201,67 @@ static void test_assignElse ()
 	keyDel (parentKey);
 	PLUGIN_CLOSE ();
 }
-int main (int argc, char ** argv)
+
+static void test_assignKeyThen()
+{
+	Key *parentKey = keyNew("user/tests/conditionals", KEY_VALUE, "", KEY_END);
```

Comment:
I'll add a shell testscript for it, the issue was related to the spec plugin, not the conditionals plugin (ok, i forgot the postgetstorage placement in the contract, but that wasn't exactly the issue)